docker/latest/Dockerfile.centos7

```diff
@@ -0,0 +1,52 @@
+FROM centos:7
+
+LABEL maintainer="info@dianomic.com"
+
+ENV ARCHITECTURE="x86_64"
+ENV OS="centos7"
+
+ARG PKG_VERSION="latest"
+
+ENV FLEDGE_PKGS_ARCHIVES_LINK="http://archives.fledge-iot.org/${PKG_VERSION}/${OS}/${ARCHITECTURE}/"
+
+# Install pre-requisites
+RUN yum update -y && \
+    yum install -y rsyslog curl wget jq centos-release-scl-rh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
+    yum install -y nginx
+
+
+# Fake service control as docker container doesn't use it
+RUN printf '#!/bin/bash \nexit 0' > /usr/bin/systemctl
+RUN chmod 755 /usr/bin/systemctl
+
+# Install Fledge repo within container
+RUN echo -e "[fledge]\n\
+name=fledge Repository\n\
+baseurl=${FLEDGE_PKGS_ARCHIVES_LINK}\n\
+enabled=1\n\
+gpgkey=http://archives.fledge-iot.org/RPM-GPG-KEY-fledge\n\
+gpgcheck=1" > /etc/yum.repos.d/fledge.repo
+
+# Install fledge
+RUN yum update -y && yum install -y fledge
+
+# Fix and install fledge-gui
+RUN rm -f /usr/share/nginx/html/index.html && \
```

Comment:
Add a comment with GUI pull request reference that this will not be required, next to the 1.9.1 releases.